docs/migrate/migrate-setup.rst

```diff
@@ -98,13 +107,15 @@ Use the updated code to view backends.
 
```

Comment:
Line 95 where is says `    # To access saved credentials for the IBM cloud channel`, I don't think we need to mention the IBM Cloud channel here since this migration guide is about going from ibmq-provider, which doesn't support IBM Cloud channel.

Reply:
We decided to leave this in